commcare-salesforce-jobs/test_jobs/TEST-Upsert-Person-Case-Update.js

```diff
@@ -124,6 +124,33 @@ fn(state => {
   };
 });
 
+query(
+  `SELECT Id, Parent_Geographic_Area__c, Parent_Geographic_Area__r.Name, Parent_Geographic_Area__r.Parent_Geographic_Area__c FROM Location__c WHERE CommCare_User_ID__c = '${dataValue(
+    'form.subcase_0.case.create.owner_id'
```

Comment:
@ritazagoni I think L129 needs to be changed from `'form.subcase_0.case.create.owner_id'` to `properties.owner_id` because it looks like `owner_id` has a different path in this [Message state](https://www.openfn.org/projects/lwala-chw-support/messages/06273cc8-9a28-7684-aa44-769a0acb4412). 

FYI, the `owner_id` for any case in CommCare is now linked to their new location hierarchy... so via owner, we can determine which Village a case belongs to. And then refer to the hierarchy to find the parent Area and Catchment (what we do via this query)